soc/st/stm32/stm32u5x/poweroff.c

```diff
@@ -12,13 +12,44 @@
 
 #include <stm32_ll_cortex.h>
 #include <stm32_ll_pwr.h>
+#include <stm32_ll_usart.h>
+#include <stm32_ll_dma.h>
+
+#if DT_HAS_COMPAT_STATUS_OKAY(st_stm32u5_dma)
+#define DT_DRV_COMPAT st_stm32_uart
+#define TURN_OFF_UART_DMA(inst)									\
+	{											\
+		USART_TypeDef *usart = (USART_TypeDef *)DT_INST_REG_ADDR(inst);			\
```

Comment:
I'd rather see an empty (escaped) line below but that not a blocking issue I guess.
```c
		USART_TypeDef *usart = (USART_TypeDef *)DT_INST_REG_ADDR(inst);			\
												\
		if (DT_INST_DMAS_HAS_NAME(inst, tx)) {						\
		...